ED/src/init/ed_params.f90

```diff
@@ -3063,6 +3110,38 @@ subroutine init_pft_alloc_params()
    b2WAI(17)    = 2.0947       ! Broadleaf
    !---------------------------------------------------------------------------------------!
 
+   !---------------------------------------------------------------------------------------!
```

Comment:
Re-defining sapwood biomass is an excellent idea because as @yjkim1028 had noted a long time ago, the original definition was incorrect (by a lot).  I do have a few concerns on this:

1. If you changed sapwood biomass, then the parameters for bdead must change too, otherwise the size-AGB allometry will be off.
2. For legacy I think this should be optional.  My suggestion to ensure back-compability is that you define this as a separate functional form.  I don't think anyone is using IALLOM=1 right now so you could claim that one unless someone says something.  Otherwise, just set a new IALLOM=4 case.

Reply:
I intentionally did not change the existing sapwood biomass calculation. Sapwood is treated as bdead in my version, because I do not want cohorts to have a large carbon pool to burn off (sapwood accounts for ~70% of total wood volume for some tropical species) under negative carbon budget.